### ISSUE TITLE
Not a complete piece of documentation

### ISSUE THREAD

**Issue by mschifter** [MicrosoftDocs/cpp-docs#306]
Document authors should ask the following question on every page: did the page teach the concept? In this case, no. There's no sample of marshaling, no syntax examples. Before you link how to "extend a library" you need to first explain the content of the library.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf10ee2b-5d6f-d821-4046-06666fa242ca
* Version Independent ID: 08850e94-03c1-5706-aecf-fa28c4b25d85
* Content: [Overview of Marshaling in C++](https://docs.microsoft.com/en-us/cpp/dotnet/overview-of-marshaling-in-cpp)
* Content Source: [docs/dotnet/overview-of-marshaling-in-cpp.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/dotnet/overview-of-marshaling-in-cpp.md)
* Product: **visual-cpp**
* GitHub Login: @mikeblome
* Microsoft Alias: **mblome**

**Comment by mikeblome** [comment]
@mschifter Agreed. I made some improvements and they should be visible in a day or two. Thanks for the feedback!
